prow/git/v2/adapter.go

```diff
@@ -78,6 +78,14 @@ func (a *repoClientAdapter) ForcePush(branch string) error {
 	return a.Repo.Push(branch)
 }
 
+func (a *repoClientAdapter) CentralPush(branch string, force bool) error {
+	return errors.New("no CentralPush implementation exists in the v1 repo client")
```

Comment:
Why not implement this?

Reply:
I don't think that we should extent the v1 client in any ways.